src/utils/helpers.js

```diff
@@ -152,6 +152,7 @@ export function buildRequestBody(body) {
 }
 
 export function buildCartItemData(
+  IsId,
```

Comment:
@yasiloghmani this function is also used by `AddPromotion`, so we should probably make `IsId` the last param in this function instead of the first (that way it can be optional) https://github.com/moltin/js-sdk/blob/a4707df10df65bc1424b7fd0d28d7cff7d48712f/src/endpoints/cart.js#L64

Also can we change it to `isId` instead of `IsId` to keep our naming format more consistent? 👍